src/Crypto/Signing/IHsmSignerService.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright 2020 De Staat der Nederlanden, Ministerie van Volksgezondheid, Welzijn en Sport.
+// Licensed under the EUROPEAN UNION PUBLIC LICENCE v. 1.2
+// SPDX-License-Identifier: EUPL-1.2
+
+using System.Threading.Tasks;
+
+namespace NL.Rijksoverheid.ExposureNotification.BackEnd.Crypto.Signing;
+
+public interface IHsmSignerService
+{
+    Task<byte[]> GetNlSignatureAsync(byte[] content);
```

Comment:
nit: maybe rename this to `GetCmsSignatureAsync`?

Reply:
Good point, done (in more places than just here though... 😅 )